### PR TITLE
Fix: Toggle Problem Between Login and Forgotten Account (#616)

### PR DESCRIPTION
**Pull request by Ram81**
@aka-jain @deshraj  please review!

### CODE REVIEW

File: frontend/src/views/web/partials/dyn-header.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
       <ul class="right hide-on-med-and-down">
         <li><a class="waves-effect waves-dark" ui-sref="home" ui-sref-active="active">Home</a></li>
        
-        <li ng-hide="isAuth"><a class="waves-effect waves-dark" ui-sref="auth.login" ui-sref-active="active">LogIn</a></li>
+        <li ng-hide="isAuth"><a class="waves-effect waves-dark" ui-sref="auth.login" ui-sref-active="active" ng-click="auth.resetForm()">LogIn</a></li>
```
Comment (aka-jain) [inline frontend/src/views/web/partials/dyn-header.html]: @Ram81 Don't use click with ui-sref as both are click events and trigger together. So use resetform function on changing of location. Check state route for location based changes.

**Comment by Ram81** [comment]
@aka-jain  Sorry for this will do changes & let you know

**Comment by Ram81** [comment]
@aka-jain  please review!

File: frontend/src/js/controllers/authCtrl.js [hunk 2]
```diff
@@ -276,6 +276,11 @@
                 vm.stopLoader();
             }
         }
+        
+        $rootScope.$on('$stateChangeStart',function () {
```
Comment (aka-jain) [inline frontend/src/js/controllers/authCtrl.js]: Have you tried it with vm, I mean replace rootscope with vm?

Comment (Ram81) [inline frontend/src/js/controllers/authCtrl.js]: @aka-jain Yeah I Tried first using vm but it doesn't work

Comment (aka-jain) [inline frontend/src/js/controllers/authCtrl.js]: That's strange, It should work. Let me check!

Comment (Ram81) [inline frontend/src/js/controllers/authCtrl.js]: @aka-jain  did it work?

Comment (aka-jain) [inline frontend/src/js/controllers/authCtrl.js]: Haven't checked it yet. Will do it today.

**Comment by Ram81** [comment]
@aka-jain  any progress with this?

**Comment by aka-jain** [comment]
@Ram81 Sorry for late responding, your code is working fine. I'm merging it.

**Pull request merged**